01-introduction.Rmd

```diff
@@ -39,49 +39,51 @@ Moreover, throughout the book we encourage you to make geographic research more
+
 The book's links to older disciplines were reflected in suggested titles for the book: *Geography with R* and *R for GIS*.
 Each has advantages.
-The former conveys the message that it's not just about spatial data: 
+The former conveys the message that it comprises much more than just spatial data: 
 non-spatial attribute data are inevitably interwoven with geometry data, and Geography is about more than where something is on the map.
 The latter communicates that this is a book about using R as a GIS, to perform spatial operations on *geographic data* [@bivand_applied_2013].
 However, the term GIS conveys some connotations (see Table \@ref(tab:gdsl)) which simply fail to communicate one of R's greatest strengths:
 its unparalleled console-based ability to seamlessly switch between geographic and non-geographic data processing, modeling and visualization tasks.
-```{r gdsl, echo=FALSE, message=FALSE}
-d = readr::read_csv("extdata/gis-vs-gds-table.csv")
-knitr::kable(x = d, caption = "Differences in emphasis between the fields of Geographic Information Systems (GIS) and Geographic Data Science (GDS).")
-```
+<!--
+"R for GIScience" or "GIScience with R" would have not implied GUI-based geoprocessing. The same goes for "Geo-data science with R"; maybe we should point out more prominently that Leeds was the birth place of geocomputation, and therefore the natural choice for the title :)
+-->
+By contrast, the term geocomputation implies reproducible and creative programming.
+Of course, (geocomputational) algorithms are powerful tools that can become highly complex.
+However, all algorithms are composed of smaller parts.
+By teaching you its foundations and underlying structure, we aim to empower you to create your own innovative solutions to geographic data problems.
 
 <!--
 All of the text below does not belong to this section, it probably belongs more to the Why R section
 
 R is an object-oriented, [functional programming language](http://adv-r.had.co.nz/Functional-programming.html) [@wickham_advanced_2014].
-Furthermore, it was specifically designed as an interactive interface to other software [@chambers_extending_2016]. 
+Furthermore, R was specifically designed as an interactive interface to other software [@chambers_extending_2016]. 
 This also includes many 'bridges' to a treasure trove of GIS software, libraries and functions.
 
 todo - add this reference to end of previous line:
 (\@ref(gis)).
 
 It is thus ideal for quickly creating 'geo-tools', without needing to master lower level languages (compared to R) such as C, FORTRAN and Java. 
-Though it is also possible for the advanced users to extend R with the power of other languages (e.g., C++ through **Rcpp** or Python through **reticulate**).
+
+This can feel like breaking free from the metaphorical 'glass ceiling' imposed by GUI-based proprietary geographic information systems (see Table \@ref(tab:gdsl) for a definition of GUI).
```

Comment:
I like this change - gets to the point quickly rather than banging on about algorithms.